app/views/devise/registrations/new.html.erb

```diff
@@ -3,6 +3,11 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
```

Comment:
Again, kindly ensure the Login screen, and the general design of your app is consistent with the [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=&).
- [ ] Layout of each screen should match the original design. Items should be properly positioned, similar to the original design
- [ ] Contents should properly fit on mobile screen versions
- [ ] Ensure that typography (fonts) used matches the original design
- [ ] Background colors of items in a layout should match the original design
- [ ] A navigation button to move to and fro the screens should be present


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/202860123-567382c6-e9da-4516-afba-95270a723fc4.png)
![image](https://user-images.githubusercontent.com/12745474/202860150-f45aca0d-8233-41c7-9b7b-7b768accd707.png)
![image](https://user-images.githubusercontent.com/12745474/202860354-2ccfc07b-0093-4c3e-ae5d-2420fab83169.png)
![image](https://user-images.githubusercontent.com/12745474/202860454-e08087b8-34c4-47b1-9a49-55b38d5b6aeb.png)
![image](https://user-images.githubusercontent.com/12745474/202860629-606a46e6-b2c4-40ca-bbce-e6b72cd68206.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/198862604-970f8265-6204-4725-b617-6edaad3469fa.png)
![image](https://user-images.githubusercontent.com/12745474/198862696-8ccc5272-9c9e-496d-8690-101f7fd39f16.png)
![image](https://user-images.githubusercontent.com/12745474/198862732-461249af-9a82-4d28-8539-8d46e0114f87.png)

Reply:
@rloterh , Style header